tests/test_modeling_common.py

```diff
@@ -2276,8 +2276,9 @@ def test_no_super_init_config_and_model(self):
     def test_checkpoint_sharding_local(self):
         model = BertModel.from_pretrained("hf-internal-testing/tiny-random-bert")
 
-        for max_size in ["50kB", "100kB", "200kB"]:
-            with tempfile.TemporaryDirectory() as tmp_dir:
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            # We use the same folder for various sizes to make sure a new save erases the old checkpoint.
+            for max_size in ["50kB", "100kB", "200kB"]:
```

Comment:
Would be down to try with the kibibytes too (`50KiB` for example)

Reply:
Sure!